### PR TITLE
Revise French docs

### PR DESCRIPTION
**Pull request by merwok**
Translated a new method and took the opportunity to go over some existing things.

@grewn0uille would like your review here!

### CODE REVIEW

File: doc/README.fr.md [hunk 1]
```diff
@@ -438,12 +439,12 @@ Dessine un triangle avec les sommets (`x1`, `y1`), (`x2`, `y2`), (`x3`, `y3`) et
...
+Copie la région de taille (`w`, `h`) de (`u`, `v`) de la banque d’image `img` (0-2) à (`x`, `y`). Si une valeur négative est mise pour `w` (ou `h`), la copie sera inversée horizontalement (ou verticalement). Si `colkey` est spécifiée, elle sera traitée comme une couleur transparente.
 
 <img src="images/blt_figure.png">
 
 - `bltm(x, y, tm, u, v, w, h, [colkey])`<br>
-Copy the region of size (`w`, `h`) from (`u`, `v`) of the tilemap `tm` (0-7) to (`x`, `y`). If negative value is set for `w` and/or `h`, it will reverse horizontally and/or vertically. If `colkey` is specified, treated as transparent color. The size of a tile is 8x8 pixels and is stored in a tilemap as a tuple of `(x in tile, y in tile)`.
+Copie la région de taill (`w`, `h`) de (`u`, `v`) de la tilemap `tm` (0-7) à (`x`, `y`). Si une valeur négative est mise pour `w` (ou `h`), la copie sera inversée horizontalement (ou verticalement). Si `colkey` est spécifiée, elle sera traitée comme une couleur transparente. La taille d’une tuile est 8x8 pixels et elle est storée dans une tilemap en tant que paire `(x dans tuile, y dans tuile)`.
```
Comment (merwok) [inline doc/README.fr.md:447]: D’autres méthodes ont `(x in tile, y in tile)`, à modifier à mon sens.
(Mais pour ça il faudrait que je comprenne la doc originelle!  est-ce que «tile» est en fait «tilemap»?)

Comment (kitao) [reply doc/README.fr.md:447]: Thank you for your confirmation.
In the latest Pyxel, basically every numbers to specify arguments for drawing functions are pixel(dot)-based. But the exception is the parameter for tiles.
Pyxel's tilemap consists of tiles which is 8x8 pixel sized unit and refers to an image banks. Tiles are represented as a tuple of referencing position (x, y). This (x, y) is the position in tile units. So if their position value change 1, referred image position change 8 pixel.
So I mentioned that (x in tile, y in tile) to notify this number is not pixel-based but tile-based.

Comment (merwok) [reply doc/README.fr.md:447]: Ah!  There could be a convention like `tilex, tiley` or `mapx, mapy` to make this clearer.  Or phrases like `tile coordinates` to differentiate from `screen coordinates`.

Comment (kitao) [reply doc/README.fr.md:447]: Thank you for your suggestion. I agree. You naming convention is clearer.
I'll replace `x in tile` to `tile_x` in every language's README soon after.
So you can keep them for now.

**Comment by kitao** [comment]
Sorry! I always overwrite READMEs immediately. Because several volunteers
often updates them when they notice some mistakes or missing parts. And it
would be difficult if I should wait for some particular one’s update. I’m
sorry if I’m offended. But please understand there is no special one for PR
of Pyxel.

2022年1月5日(水) 0:01 Éric ***@***.***>:

**Comment by grewn0uille** [comment]
Hi!
I read your PR @merwok and I agree on the translation :).
I just opened https://github.com/kitao/pyxel/pull/359 to add a missing "e" and translate a title which was in English.

**Comment by merwok** [comment]
Ah no offense!  Just wanted to understand 🙂 
I think you could trust translators to review open tickets / PRs and trust the draft status when set by a contributor.

**Pull request merged**